packages/blade/src/components/SideNav/_decisions/decisions.md

```diff
@@ -0,0 +1,545 @@
+| titleSuffix | Slot after the title to add Badge, Counter                                                                                   | JSX                           |                                                              |
+| tooltip     | Object with props that are forwarded to tooltip                                                                              | TooltipProps                  | undefined                                                    |
+| children    | SideNavLink children slot. Items inside children turn into next level item with parent as a trigger                          | JSX                           | undefined                                                    |
+
+#### Example Nav Links
+
+<table>
+
+<tr><td>Code</td><td>Preview</td></tr>
+
+<tr>
+<td>
+
+<!-- prettier-ignore -->
+```jsx
+<SideNavLink
+  as={NavLink}
+  icon={BillIcon}
+  title="Vendor Payments"
+  href="/vendor-payments"
+/>
+```
+
+</td>
+<td><img src="navlink-eg-1.png" width="200px" /></td>
+</tr>
+
+<tr>
+<td>
+
+<!-- prettier-ignore -->
+```jsx
+<SideNavLink
+  as={NavLink}
+  icon={LayoutIcon}
+  title="L1 Item Name"
+  href="/l1-item"
+  // This will be controlled on consumer
+  // depending on react-router active state
+  isActive={true}
+/>
+```
+
+</td>
+<td><img src="navlink-eg-active.png" width="200px" /></td>
+</tr>
+<tr>
+<td>
+
+<!-- prettier-ignore -->
+```jsx
+<SideNavLink
+  as={NavLink}
+  icon={PayoutIcon}
+  title="Create Payouts"
+  href="/payouts"
+  trailing={
+    <Button
+      href="/payouts/create"
+      icon={PlusIcon}
+      size="xsmall"
+      variant="tertiary"
+    />
+  }
+/>
+```
+
+</td>
+<td><img src="navlink-eg-2.png" width="200px" /></td>
+</tr>
+
+<tr>
+<td>
+
+<!-- prettier-ignore -->
+```jsx
+<SideNavLink
+  as={NavLink}
+  icon={LayoutIcon}
+  title="L1 Item Name"
+  href="/new-item-link"
+  titleSuffix={
+    <Badge color="positive">New</Badge>
+  }
+/>
+```
+
+</td>
+<td><img src="navlink-eg-3.png" width="200px" /></td>
+</tr>
+
+<tr>
+
+<td>
+
+<!-- prettier-ignore -->
+```jsx
+<SideNavLink
+  as={NavLink}
+  icon={LayoutIcon}
+  title="L1 Item Name"
+  href="/new-item-link"
+  tooltip={{
+    content: "Action Name (Cmd + P)"
+  }}
+/>
+```
+
+</td>
+<td><img src="navlink-eg-4.png" width="240px" /></td>
+</tr>
+
+<tr>
+
+<td>
+
+<!-- prettier-ignore -->
+```jsx
+<SideNavLink
+  as={NavLink}
+  icon={ArrowUpRightIcon}
+  title="Create Payout"
+  href="/payouts"
+  trailing={
+    <Tooltip content="Action Name (Cmd + P)">
+      <Button
+        href="/payouts/create"
+        icon={PlusIcon}
+        size="xsmall"
+        variant="tertiary"
+      />
+    </Tooltip>
+  }
+/>
+```
+
+</td>
+<td><img src="navlink-eg-tooltip.png" width="300px" /></td>
+</tr>
+
+</table>
+
+### SideNavLevel
```

Comment:
I hope we will handle the edge case where people try to add L4 sidenav level and throw some meaningful error

Reply:
yup added in implementation